README.md

```diff
@@ -30,3 +30,10 @@ Licensing and Pricing options are customized to your usage and requirements. For
 - Printing
 - Security - Password, encrypt, watermark
 - Search for and replace or edit text
+
+## Instructions for trying out the samples
+- Clone this repository to your system (64-bit Intel Windows, 64-bit Intel Linux, or Apple Silicon)
+- Select a sample you would like to try
+- Open the folder containing the sample in [IntelliJ IDEA](https://www.jetbrains.com/idea/)
+- From the Maven tab in the right sidebar, run the "Compile" goal
```

Comment:
I couldn't find a way to make the IDE process the resources automatically (there's a button you can press in the Maven pane that will do that), but if we make changes like this to the run configurations, then all the user/evaluator has to do is to press the Run button. The IDE will run the `process-resources` phase automatically, and after it's run once, it's fast (the dependencies plugin can tell if the dependencies are already unpacked).

```diff
diff --git a/ContentCreation/AddElements/.idea/runConfigurations/AddElements.xml b/ContentCreation/AddElements/.idea/runConfigurations/AddElements.xml
index 265fb74..19777e8 100644
--- a/ContentCreation/AddElements/.idea/runConfigurations/AddElements.xml
+++ b/ContentCreation/AddElements/.idea/runConfigurations/AddElements.xml
@@ -7,6 +7,7 @@
     <module name="AddElements" />
     <option name="VM_PARAMETERS" value="-Djava.library.path=./target/lib" />
     <method v="2">
+      <option name="Maven.BeforeRunTask" enabled="true" file="$PROJECT_DIR$/pom.xml" goal="process-resources" />
       <option name="Make" enabled="true" />
     </method>
   </configuration>
```

(This is the button that generates the sources and puts the libraries in place)

<img width="467" alt="image" src="https://github.com/datalogics/apdfl-java-maven-samples/assets/89971/c7ddc283-221d-4ab7-883c-8fec312154d1">

Reply:
I like this idea, but I'll defer it to another pull request.